### ISSUE TITLE
throw() in function signature breaks parser

### ISSUE THREAD

**Issue by aemsle** [slynch8/10x#2005]
When using the keyword throw() in a function signature:
 ```cpp
class ClassWithNoThrow
{
public:
    void DoSomething() const throw() { std::cout << "DoSomethingCalled!\n"; }
};
```
the parser no longer highlights the function properly, however it will still recognize it for autocomplete and goto. 

10x Behavior: 
![10x_dnd22OwUrA](https://github.com/slynch8/10x/assets/44575716/50b33f3b-6759-42b7-ba84-0967b61442f2)

VS+VAX Behavior: 
![devenv_BUVwnRQmVo](https://github.com/slynch8/10x/assets/44575716/35527ffd-5de1-41e4-a940-ba2bf10643bc)

**Comment by slynch8** [comment]
fixed (1.0.74)

**Comment by aemsle** [comment]
Unsure if this is a regression or not but this seems to only work when the function body is inline in the header. 
![10x_Jr0Y3rOSqC](https://github.com/slynch8/10x/assets/44575716/55591d20-1008-494b-92b7-424914a3b1b5)
